### PR TITLE
Delete pv if it exists before creating

### PR DESCRIPTION
**Pull request by sankarpn**
This PR will address test failures caused existing pv issues when running tests. The change deletes the pv if it exists before creating the pv with same name.

This is what will happen with this change.

`<03-11-2022 07:05:43> <INFO> <oracle.weblogic.kubernetes.utils.PersistentVolumeUtils getUniquePvOrPvcName> <Creating unique pv|pvc name myintrodomain-pv-wuowtp>
<03-11-2022 07:05:43> <INFO> <oracle.weblogic.kubernetes.utils.PersistentVolumeUtils getUniquePvOrPvcName> <Creating unique pv|pvc name myintrodomain-pvc-hcqokk>
<03-11-2022 07:05:43> <INFO> <oracle.weblogic.kubernetes.utils.PersistentVolumeUtils createPV> <deleting persistent volume pvName myintrodomain-pv-wuowtp if it exists>
<03-11-2022 07:05:43> <WARNING> <oracle.weblogic.kubernetes.actions.impl.primitive.Kubernetes deletePv> <Failed to delete persistent volume 'myintrodomain-pv-wuowtp' with HTTP status code: 404>
<03-11-2022 07:05:45> <INFO> <oracle.weblogic.kubernetes.utils.CommonTestUtils$1 conditionEvaluated> <Completed: pv myintrodomain-pv-wuowtp to be deleted (elapsed time 2,004 ms)>
<03-11-2022 07:05:45> <INFO> <oracle.weblogic.kubernetes.utils.PersistentVolumeUtils createPV> <creating persistent volume for pvName myintrodomain-pv-wuowtp, domainUid: myintrodomain, className: ItIntrospectVersion>`

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9225/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9219/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9211/
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9210/

### CODE REVIEW

File: kindtest.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/bash -x
```
Comment (anpanigr) [inline kindtest.sh:1]: Why this is needed ?

**Comment by vanajamukkara** [comment]
Please update with Jenkins results

**Review by vanajamukkara — APPROVED** [review]
LG

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java [hunk 2]
```diff
@@ -68,8 +68,8 @@
   private static String domainNamespace = null;
   private static int replicaCount = 2;
   private static final String domainUid = "mii-custom-ssl";
-  private static String pvName = domainUid + "-pv"; 
-  private static String pvcName = domainUid + "-pvc"; 
+  private static final String pvName = domainUid + "-pv"; // name of the persistent volume
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java:71]: Why do not we use getUniquePvOrPvcName()  here ?

File: kindtest.sh (hunk 1 above)
Comment (sankarpn) [reply kindtest.sh]: I was trying to debug the docker registry issue. removed it.

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java (hunk 2 above)
Comment (sankarpn) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiCustomSslStore.java:71]: made changes to create unique pv|pvc names

**Review by anpanigr — APPROVED** [review]
lgtm

**Pull request merged**